docs/environment.yml

```diff
@@ -2,7 +2,7 @@ name: intake-esm-dev
 channels:
   - conda-forge
 dependencies:
-  - dask
+  - dask>=2.2
```

Comment:
This should already bring in fsspec. Let us know if this is not the case.

Reply:
You are right.. I just fixed it! Thank you